website/docs/components/breadcrumb/partials/code/how-to-use.md

```diff
@@ -24,7 +23,7 @@ A few parameters were omitted for clarity.
 Add the correct `@route/@models/@model/@query` parameter to each Breadcrumb Item.
 
 ```handlebars
-<Hds::Breadcrumb>
+<Hds::Breadcrumb aria-label="breadcrumbs with routing params">
```

Comment:
I would ask you to consider this: We expect our examples to work, we expect our CSS and JavaScript to be correct, why would Accessibility be any different? 

1. If I am a user with a screen reader, and I visit this page, I am presented with a list of breadcrumbs, with no way to differentiate among them. So it becomes a non-equitable discoverability issue: while a user who can see the screen can quickly differentiate among examples and copy the example that is appropriate for them, the user with AT has to go through each example and remember which examples are there, then try to find the one that they want to copy. 
2. I do not think our consumers will be confused by the presence of different aria-labels. The presence of unique labels will reinforce the Success Criteria and provide supplementary learning _through the examples_. 

I hope you will re-consider your point of view.

Reply:
When consumers copy and paste the code snippet into their project, do we expect them to keep the `aria-label`, or is this something they would likely remove? 

If it's something they'll remove, is there a way to attach the `aria-label` to the code snippet block (container) instead of directly in the component itself?

Reply:
@heatherlarsen @KristinLBradley there's already an `aria-label` on the component itself (default is `aria-label="breadcrumb"`) so that would take precedence.